tests/components/wunderground/test_sensor.py

```diff
@@ -123,7 +128,7 @@ async def test_sensor(hass, aioclient_mock):
     state = hass.states.get("sensor.pws_precip_1d_in")
     assert state.state == "0.03"
     assert state.name == "Precipitation Intensity Today"
-    assert state.attributes["unit_of_measurement"] == LENGTH_INCHES
+    assert state.attributes[ATTR_UNIT_OF_MEASUREMENT] == LENGTH_INCHES
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)